mxcubecore/model/lims_session.py

```diff
@@ -16,6 +16,18 @@
 
 
 class Proposal(BaseModel):
+    """Proposal with key identifying details.
+
+    Attributes:
```

Comment:
```suggestion
    Parameters:
```

Alright, seems like using `Parameters` as docstring section title is a simple and easy fix. It will avoid having everything in double.

I am still not 100% convinced it is a very good solution, but more than good enough for now. : )

Reply:
I think it should be fine now